lake/ztests/query-parse-error.yaml

```diff
@@ -34,34 +34,34 @@ outputs:
   - name: stderr
     data: |
       =1=
-      error parsing Zed at column 11:
+      error parsing Zed (line 1, column 11):
```

Comment:
This looks like a purely cosmetic change. Can we save it in a separate PR?

Reply:
Not really since with semantic errors you have different error messages that may not syntactically work in the "append this to a sentence" way that errors are currently formatted.